### ISSUE TITLE
Found 0 validated image filenames

### ISSUE THREAD

**Issue by loaialamro** [loaialamro/Keras-FlowFromDataframe#1]
Hi

I tried to use FlowFromDataframe but it cannot detect the validation images and I get this

Found 162770 validated image filenames.

Found 0 validated image filenames.

Can anyone help me please

My code is here:

traindf=pd.read_csv('E:/color.csv')
columns=["Black", "White", "Red", "Blue", "Green", "Brown", "Yellow"]
datagen=ImageDataGenerator()

train_generator=datagen.flow_from_dataframe(
dataframe=traindf,
directory="E:/Experimental_Data",
x_col="image_id",
y_col=columns,
subset="training",
batch_size=32,
seed=42,
shuffle=True,
class_mode="raw",
target_size=(178,218))

test_datagen=ImageDataGenerator()
valid_generator=test_datagen.flow_from_dataframe(
dataframe=traindf,
directory="E:/Experimental_Data",
x_col="image_id",
y_col=columns,
subset="validation",
batch_size=32,
seed=42,
shuffle=True,
class_mode="raw",
target_size=(178,218))
